src/services/club.py

```diff
@@ -179,3 +181,28 @@ async def handover(
                 )
         await new_pres.add_roles(role_pres, reason=f"Passation du club : {club.name}")
         await new_treso.add_roles(role_treso, reason=f"Passation du club : {club.name}")
+        category = utils.get(guild.categories, id=club.category_id)
+        if category.name == club.name + " [inactif]":
+            await category.edit(position=4, name=club.name)
+
+        await category.edit(position=4)
```

Comment:
```suggestion
        if category.name.endswith("[inactif]"):
            await category.edit(name=club.name)
        highest_inactive = discord.utils.find(
            lambda c: c.name.endswith("[inactif]"),
            sorted(guild.categories, key=lambda c: c.position),
        )
        await category.move(above=highest_inactive)
```
Rien ne garantit qu'on gardera 3 catégories non-club, et qu'il n'y en aura pas une en plus ou en moins.
Donc c'est possiblement plus sûr de modifier la logique pour bouger la catégorie au-dessus du club inactif le plus haut.

Ca a en plus l'effet que les clubs intermittents, qui tiennent un semestre puis redeviennent inactifs avant d'éventuellement être repris l'année d'après, se retrouveront pas au-dessus de clubs plus actifs dans la liste des clubs.